mono/metadata/w32process-win32.c

```diff
@@ -399,14 +399,12 @@ process_unquote_application_name (gchar *appname)
 static gchar*
 process_quote_path (const gchar *path)
 {
-	gchar *res = g_shell_quote (path);
-	gchar *q = res;
-	while (*q) {
-		if (*q == '\'')
-			*q = '\"';
-		q++;
-	}
-	return res;
+	size_t len = strlen (path);
```

Comment:
Do we lose anything dropping the call to `g_shell_quote`?

Reply:
No I think it was purposefully avoided when the original commit was implemented.